tests/Datagrid/ProxyQueryTest.php

```diff
@@ -60,6 +60,7 @@ protected function setUp(): void
 
     protected function tearDown(): void
     {
+        // @phpstan-ignore-next-line
```

Comment:
why is this one needed?

Reply:
Cause we're setting null to a variable which is not nullable.

Maybe a PR could be made to phpstan-phpunit

Reply:
I think I have a better solution.
Using `unset($this->em);`